package.nls.ko.json

```diff
@@ -0,0 +1,28 @@
+{
+	"plantuml.exportCurrent.title": "현재 다이어그램 내보내기",
+	"plantuml.exportDocument.title": "현재 다이어그램 파일 내보내기",
+	"plantuml.exportWorkspace.title": "작업공간의 다이어그램 내보내기",
+	"plantuml.preview.title": "현재 다이어그램 미리보기",
+	"plantuml.URLCurrent.title": "현재 다이어그램에 대한 URL 생성하기",
+	"plantuml.URLDocument.title": "현재 다이어그램 파일에 대한 URL 생성하기",
+	"plantuml.extractSource.title": "그림에서 PlantUML 다이어그램 소스 추출하기",
+	"plantuml.configuration.configTitle": "PlantUML 환경설정",
+	"plantuml.configuration.exportFormat": "내보내기 형식. 매전 내보내기할 때마다 선택하려면 비워두십시오.",
+	"plantuml.configuration.jar": "plantuml.jar의 다른 위치. 통합 jar를 쓰려면 비워두십시오.",
+	"plantuml.configuration.diagramsRoot": "모든 다이어그램 파일들을 저장할 위치 (작업공간의 상대 폴더).",
+	"plantuml.configuration.fileExtensions": "내보내기 찾을 때 사용할 파일 확장자. 특히 작업공간 설정에서, 소스코드 파일의 다이어그램을 내보내기할 때 \".java\" 같은 사용자의 확장를 추가할 수 있습니다.",
```

Comment:
I think this paragraph is hard to understand. What about this?
```
내보내기 할 파일의 확장자들. ~~
```

Reply:
Updated a bit, how about now?

Reply:
LGTM 😄